test/python/transpiler/test_remove_final_reset.py

```diff
@@ -15,10 +15,11 @@
 import unittest
 
 from qiskit import QuantumRegister, QuantumCircuit
+from qiskit.passmanager.flow_controllers import DoWhileController
 from qiskit.transpiler import PassManager
 from qiskit.transpiler.passes import RemoveFinalReset, DAGFixedPoint
 from qiskit.converters import circuit_to_dag
-from qiskit.test import QiskitTestCase
+from test import QiskitTestCase  # pylint: disable=wrong-import-order
```

Comment:
FWIW, you could make pylint happy if you added a new line between the `qiskit` import section and this import.